components/Containers/Sidebar/index.tsx

```diff
@@ -1,9 +1,11 @@
+'use client';
+
 import { useTranslations } from 'next-intl';
 import type { ComponentProps, FC } from 'react';
 
 import SidebarGroup from '@/components/Containers/Sidebar/SidebarGroup';
 import WithRouterSelect from '@/components/withRouterSelect';
-import { useClientContext } from '@/hooks/react-server';
+import { usePathname } from '@/navigation.mjs';
```

Comment:
Please use the client context hook which can be invoked from the server side. No need to make this client-side.

Or is there any reason for making this component client-side? Regardless, we also have a client-side hook for client-context, I discourage using the navigation.mjs file.

Reply:
Since I imported this into the base layout, I had to make it a client component. Same for Navbar and ProgressionSidebar.

Reply:
I forgot to mention that the sidebar is out of the scope of the matter context provider because it's in the base layout now. So using the client-side context hook won't work.